packages/core/backend/sirius-components-collaborative/src/main/java/org/eclipse/sirius/components/collaborative/api/IRepresentationPersistenceService.java

```diff
@@ -25,6 +25,8 @@ public interface IRepresentationPersistenceService {
 
     void save(ICause cause, IEditingContext editingContext, IRepresentation representation);
 
+    void save(ICause cause, IEditingContext editingContext, String representationId, String representationContent, String representationKind);
```

Comment:
You have a new feature in Sirius Web and you need to add a new API to one of the most central API deep down in Sirius Components just for your need? That seems very very odd.

Reply:
Done